components/openvino/install.sh

```diff
@@ -23,12 +23,18 @@ tar -xzf `ls | grep "openvino_toolkit"`
 cd `ls -d */ | grep "openvino_toolkit"`
 
 apt-get update && apt-get install -y sudo cpio && \
- ./install_cv_sdk_dependencies.sh && SUDO_FORCE_REMOVE=yes apt-get remove -y sudo
+ if [ -f "install_cv_sdk_dependencies.sh" ]; then ./install_cv_sdk_dependencies.sh; \
+ else echo ./install_openvino_dependencies.sh; fi && SUDO_FORCE_REMOVE=yes apt-get remove -y sudo
```

Comment:
echo?

Reply:
@alalek Yes, I have already found the issue.